underfs/hdfs/src/main/java/alluxio/underfs/hdfs/HdfsUnderFileSystem.java

```diff
@@ -283,11 +282,11 @@ public long getSpace(String path, SpaceType type) throws IOException {
     if (mFileSystem instanceof DistributedFileSystem) {
       switch (type) {
         case SPACE_TOTAL:
-          return ((DistributedFileSystem) mFileSystem).getDiskStatus().getCapacity();
+          return mFileSystem.getStatus().getCapacity();
```

Comment:
This needs to be reverted to keep compatibility with hadoop 1 - could you suppress the warning and add an explanation?